### ISSUE TITLE
Tab suggestion replaces function

### ISSUE THREAD

**Issue by FunghiFannoBene** [microsoft/vscode-cpptools#11819]
### Environment

- OS and Version: Ubuntu 22.04
- VS Code Version: Latest
- C/C++ Extension Version:Latest 
- If using SSH remote, specify OS of remote machine: no

### Bug Summary and Steps to Reproduce

Bug Summary:
add class(suggestion) to function using tab
for example:
void function();
void Wron(tab)function(); = void WrongCat()

expected: void WrongCatfunction();
to be completed as WrongCat::function();

C++ extention override vscode "insert mode" and uses "replace" for functions.

### Configuration and Logs

```shell
Just replacement insead of insert mode, cannot be edited from  c_cpp_properties.json
```

**Comment by sean-mcmanus** [comment]
@FunghiFannoBene This should be fixed 1.19.1: https://github.com/microsoft/vscode-cpptools/releases/tag/v1.19.1 after you add settings

```
            "[cpp]": {
                "editor.suggest.insertMode": "insert",
            },
            "[cuda-cpp]": {
                "editor.suggest.insertMode": "insert",
            },
            "[c]": {
                "editor.suggest.insertMode": "insert",
            }
```

Does that work for you?

**Comment by FunghiFannoBene** [comment]
It worked, sorry for duplicate couldn't find a solution
